webknossos/webknossos/dataset/view.py

```diff
@@ -203,24 +435,107 @@ def get_view(
         """
         if read_only is None:
             read_only = self.read_only
-        assert (
-            read_only or read_only == self.read_only
-        ), "Failed to get subview. The calling view is read_only. Therefore, the subview also has to be read_only."
+        else:
+            assert (
+                read_only or not self.read_only
+            ), "Failed to get subview. The calling view is read_only. Therefore, the subview also has to be read_only."
+
+        current_mag_size: Optional[Vec3IntLike]
+        mag1_size: Optional[Vec3IntLike]
+
+        if offset is None:
+            if size is None:
+                assert (
+                    relative_offset is None and absolute_offset is None
+                ), "You must supply a size, when using get_view with an offset."
```

Comment:
again offset is None at this place

Reply:
Same as above, I think that's fine. Maybe the confusion is that offset, relative_offset and absolute_offset can be given separately?

Reply:
Yes, that probably contributed to my confusion :sweat_smile: